lanehunter-dev/refactor-tractor
# Css bug fixes

#### What's this PR do?
Fixes banner layout and a bug causing banner to shift. Adds error messages depending on which 
page the user tries to navigate to
#### Where should the reviewer start?
banner.scss
domUpdates/scripts to understand error messages
#### How should this be manually tested?
try to enter a page without saving a recipe first, then see if the error message is still there after returning to home page
#### Questions:
 Need media queries for error messages
